packages/smart-wallet/src/smartWallet.js

```diff
@@ -101,12 +95,10 @@ const mapToRecord = map => Object.fromEntries(map.entries());
  *   walletStorageNode: StorageNode,
  * }} UniqueParams
  *
+ * @typedef {Pick<MapStore<Brand, BrandDescriptor>, 'has' | 'get' | 'values'>} ReadOnlyMapStore
```

Comment:
`ReadOnlyMapStore` would be generic on the key/value types. Please pick a name that's specific to the Brand/BrandDescriptor mapping or this use case. `BrandDescriptorRegistry`?